### ISSUE TITLE
jetlag

### ISSUE THREAD

**Issue by zazazouthecat** [eikek/sharry#634]
Hi,
First thx for sharry it's a very good self-hosted tiny share file, good good job !!
I have a +1 hour jet lag problem, but I don't understand why.
summer time, winter time in France
![image](https://user-images.githubusercontent.com/30299307/142421306-fb2f039a-aa61-4e56-a506-e2e1fbfc43ea.png)

I have to look for a solution but I am at a dead end.
an idea?
Thank you.

**Comment by eikek** [comment]
Thank you @zazazouthecat ! I'm afraid this is the inability of the frontend to know your local time zone :( it presents the UTC time. If your local time is UTC+1 this would explain it?

**Comment by zazazouthecat** [comment]
Hi thank for you aswer,
That right actualy it's UTC +1 in France.
Maybe in a future update add the timezone in sharry.conf?

**Comment by eikek** [comment]
Yes, sure! I'd like to have it a client setting, but a fallback timezone in the config file is also a good thing. We can leave this issue open

**Comment by erikgieseler** [comment]
Is there any news on this?
It would be nice if you could set the time zone yourself.

**Comment by eikek** [comment]
Fixed in #1819
